#!/usr/bin/env python
PACKAGE = "bartender_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("alpha_1", double_t, 0, "First task coefficient",    8, 1,   20)
gen.add("alpha_2",    double_t,    0, "Second task coefficient",  0.4, 0,   10)
gen.add("second_task",   bool_t,   0, "Using gravity task",  True)

exit(gen.generate(PACKAGE, "bartender_control", "control"))